### PR TITLE
Update: `BandicamCompany.Bandicam` version 5.2.0.1855

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change? --- Yes, there are duplicates
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20913)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like a duplicate of PR #20789.

**Comment by vedantmgoyal2009** [comment]
> * [x]  Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
> * [x]  Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change? --- Yes, there are duplicates
> * [x]  Have you validated your manifest locally with `winget validate --manifest <path>`?
> * [x]  Have you tested your manifest locally with `winget install --manifest <path>`?
> * [x]  Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
> 
> Note: `<path>` is the name of the directory containing the manifest you're submitting.
> 
> ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20913)

See it again, I have already mentioned.

**Comment by ItzLevvie** [comment]
PR will run into the same issues as other PRs because it needs to be checked by an engineer.

**Comment by vedantmgoyal2009** [comment]
If it gets some error, I will close it.

**Comment by quhxl** [comment]
It will get errors. Just close it or maybe it will mess up the other PRs.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Check https://github.com/microsoft/winget-pkgs/pull/20913#issuecomment-880050079, it's missing some of the metadata from https://github.com/microsoft/winget-pkgs/pull/20789

**Comment by vedantmgoyal2009** [comment]
@ItsLevvie Now fine?

File: manifests/b/BandicamCompany/Bandicam/5.2.0.1855/BandicamCompany.Bandicam.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+#  Scope: 
+  InstallerUrl: https://dl.bandicam.com/bdcamsetup.exe
```
Comment (ItzLevvie) [inline manifests/b/BandicamCompany/Bandicam/5.2.0.1855/BandicamCompany.Bandicam.installer.yaml:16]:
```suggestion
  InstallerUrl: https://dl.bandicam.com/old/BDCAMSETUP_ENG_5_2_0_1855.EXE
```

File: manifests/b/BandicamCompany/Bandicam/5.2.0.1855/BandicamCompany.Bandicam.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/b/BandicamCompany/Bandicam/5.2.0.1855/BandicamCompany.Bandicam.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**